force-app/main/default/lwc/pubsub/README.md

```diff
@@ -0,0 +1,9 @@
+** Looking for the `c/pubsub` component? **
+
+As of the Summer '20 major release, the Lightning message service (LMS) is now Generally Available (GA).
```

Comment:
I suggest adding a link to the LWC LMS documentation:
https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.use_message_channel

Reply:
Yep. That's a good idea. I seem to recall you mentioning that, but I may have missed it when I rescanned our slack discussion. Thanks!